fairmq.sh

```diff
@@ -0,0 +1,79 @@
+package: FairMQ
```

Comment:
FairMQ should be only built using the appropriate defaults, have a look at the `valid_defaults:` section in O2 for instance (FairLogger is a mere dependency so don't add it there).

Reply:
Will look into this later or tomorrow